modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -229,6 +236,34 @@ export default class Ecdsa {
     return shares;
   }
 
+  /**
+   * Derives a child common keychain from common keychain
+   *
+   * @param commonKeychain - common keychain as a hex string
+   * @param path - bip32 path
+   * @return {string} derived common keychain as a hex string
+   */
+  deriveUnhardened(commonKeychain: string, path: string): string {
+    if (Ecdsa.hdTree === undefined) {
+      throw new Error("Can't derive key without HDTree implementation");
+    }
+
+    const keychain = Buffer.from(commonKeychain, 'hex');
+
+    const derivedPublicKeychain = Ecdsa.hdTree.publicDerive(
+      {
+        pk: bigIntFromBufferLE(keychain.slice(0, 32)),
+        chaincode: bigIntFromBufferBE(keychain.slice(32)),
+      },
+      path
```

Comment:
Kindly add sufficient unit test cases for this new function [here](https://github.com/BitGo/BitGoJS/blob/BG-61874-chain-fi-inc-forwarder-version-issue/modules/account-lib/test/unit/mpc/tss/ecdsa/ecdsa.ts#L0-L1)